lib/tenants/src/dependency.py

```diff
@@ -105,7 +116,8 @@ def decode_rs256(self, token: str) -> Dict[str, Any]:
                 status_code=status.HTTP_401_UNAUTHORIZED,
                 detail="Token is expired!",
             )
-        except (jwt.PyJWTError, jwt.exceptions.DecodeError):
+        except (jwt.PyJWTError, jwt.exceptions.DecodeError) as err:
+            logger.exception(err)
```

Comment:
Same as previous comment
```suggestion
            logger.exception("Cannot decode invalid rs256 token")
```